db/tag.c

```diff
@@ -6716,9 +6716,27 @@ void freedb_int(dbtable *db, dbtable *replace)
         free(db->ixsql);
     }
 
-    for (i = 0; i < db->n_constraints; i++) {
+    if (db->rev_constraints) {
```

Comment:
check and see if we also want to free sub-fields.

Reply:
The `constraint_t *` pointers in `rev_constraints` contains pointers owned by the referring tables so no need to free here.